CMakeLists.txt

```diff
@@ -283,6 +283,18 @@ elseif(CMAKE_COMPILER_IS_GNUCXX)
   set(CMAKE_CXX_FLAGS
     "${CMAKE_CXX_FLAGS} -fsigned-char")
 
+# MSVC **MUST** come before  the Clang check, as ClangCL flags as MSVC but still matches as Clang :)
```

Comment:
Extra space between 'before  the'.

"as ClangCL flags as MSVC" is a little awkward, can you reword?

Reply:
Should be fixed now